apps/web/next.config.mjs

```diff
@@ -11,7 +11,7 @@ const nextConfig = {
   pageExtensions: ['js', 'jsx', 'mdx', 'ts', 'tsx'],
   // Optionally, add any other Next.js config below
   images: {
-    domains: ['images.unsplash.com', 'rstr.in', 'images.pexels.com'],
+    domains: ['images.unsplash.com', 'rstr.in', 'images.pexels.com', 'storage.googleapis.com'],
```

Comment:
I think this might be too permissive. We should move to `remotePatterns` and specify the Makeswift bucket here specifically. More here: https://nextjs.org/docs/app/api-reference/components/image#remotepatterns